src/backend/joanie/tests/core/test_models_certificate.py

```diff
@@ -187,3 +188,31 @@ def test_models_certificate_verification_uri(self):
                 )
             else:
                 self.assertIsNone(certificate.verification_uri)
+
+    def test_models_certificate_get_document_context_missing_logo_raise_logger_error(
+        self,
+    ):
+        """
+        When getting the document context for a certificate but the organization does
+        not have a logo stored, it should trigger a logger error with the 'id' of the
+        organization concerned
+        """
+        organization = factories.OrganizationFactory(
+            title="University X",
+            representative="Joanie Cunningham",
+            logo=None,
+        )
+        course = factories.CourseFactory(organizations=[organization])
+        product = factories.ProductFactory(
+            courses=[course],
+            title="Graded product",
+        )
+        order = factories.OrderFactory(product=product, organization=organization)
+        certificate = factories.OrderCertificateFactory(order=order)
```

Comment:
This can be simplified by removing useless arguments.
```suggestion
        organization = factories.OrganizationFactory(logo=None)
        course = factories.CourseFactory(organizations=[organization])
        product = factories.ProductFactory(courses=[course])
        certificate = factories.OrderCertificateFactory(
            order__product=product,
            order__organization=organization,
        )
```

Reply:
it's much more concise ! thank you !